### PR TITLE
Fetch static viz code changes while running in the dev mode

### PR DESCRIPTION
**Pull request by alxnddr**
## Changes

FE engineers mostly use `yarn dev` to run everything or `clojure -M:run` to run the backend separately locally for development. When working on static viz in order to test updated code engineers needed to rebuild the static viz bundle and then rerun their backend which is not great from the developer experience perspective. This PR adds the `yarn build-static-viz:watch` command to watch for static viz files changes and update the bundle. Also, it runs as a part of `yarn dev`.

I'm not using webpack `--mode=development` because it still minimizes the code. And I'm not using `--mode=none` which does not minimize the code but it creates a bundle with globals such as `process` which does not exist in the static viz execution environment.

## How to verify

- Run `yarn dev`
- Add a progress bar visualization to a dashboard
- Subscriptions -> Email -> Send a test email
- Open `frontend/src/metabase/static-viz/components/ProgressBar/utils.ts` and change any text that was visible in the test email
- Send a test email again

## Demo

<video src="https://user-images.githubusercontent.com/14301985/196790877-19e68572-9139-4b96-85d5-cc894acf6906.mov" />

### CODE REVIEW

File: src/metabase/pulse/render/js_svg.clj [hunk 1]
```diff
@@ -30,16 +31,19 @@
...
 
-(defn- static-viz-context
-  "Load the static viz js bundle into a new graal js context."
-  []
-  (load-viz-bundle (js/context)))
-
-(def ^:private ^Context context
-  "Javascript context suitable for evaluating the charts. It has the chart bundle and the above `src-api` in its
-  environment suitable for creating charts."
+(def ^:private ^Context static-viz-context-delay
```
Comment (metamben) [inline src/metabase/pulse/render/js_svg.clj:34]: Does this type hint really help? Shouldn't it be on `context` below?

File: test/metabase/pulse/render/js_svg_test.clj [hunk 2]
```diff
@@ -47,7 +47,7 @@
     (is (.hasAttribute line "fill-opacity"))
     (is (= (.getAttribute line "fill-opacity") "0.0"))))
 
-(def ^Context context (delay (#'js-svg/static-viz-context)))
+(def ^Context context (@#'js-svg/context))
```
Comment (metamben) [inline test/metabase/pulse/render/js_svg_test.clj:50]:
```suggestion
(def ^Context context (#'js-svg/context))
```
I think the idiomatic way is just calling the var. In this form one might think that delays are involved.

**Review by metamben — APPROVED** [review]
Looks good! 👍

File: src/metabase/pulse/render/js_svg.clj (hunk 1 above)
Comment (noahmoss) [reply src/metabase/pulse/render/js_svg.clj:34]: Yep, good catch

File: webpack.static-viz.config.js [hunk 3]
```diff
@@ -47,4 +49,7 @@ module.exports = {
       "metabase-lib": LIB_SRC_PATH,
     },
   },
+  optimization: {
+    minimize: !shouldDisableMinimization
```
Comment (ranquild) [inline webpack.static-viz.config.js:53]: @alxnddr you could use `env.WEBPACK_WATCH` to disable minification in that case https://webpack.js.org/api/cli/#environment-options

**Pull request merged**